src/hillslope_method.R

```diff
@@ -35,7 +35,8 @@ hillslope_tool <- function(hillslope_nr, li_spatial, plot_2d_catena=FALSE, plot_
   # safty check if hillslope_nr is set to zero
   if(hillslope_nr == 0) stop("---Do not use hillslope_nr = 0---")
   
-  hill <- hillslope_as_pts[hillslope_as_pts[,3]==hillslope_nr,]
+  if(hillslope_nr == -1) hill <- hillslope_as_pts # Untested - Ashish 06.08.2024

```

Comment:
Why do you check for `hillslope_nr == -1` here? The `hillslope_nr` comes from the parameters and can be anything, right?

Reply:
I just found out from `tool.yml` that `hillslope_nr = -1` means that the entire hillslope is used, maybe add this as a comment here.